lib/tasks/data.rake

```diff
@@ -0,0 +1,219 @@
+      	'National Offshore Petroleum Safety and Environmental Management Authority'
+      ],
+      'Department of Infrastructure and Regional Development' => [
+      	'Infrastructure Australia',
+      	'Airservices Australia',
+      	'Australian Maritime Safety Authority',
+      	'Australian Rail Track Corporation Limited',
+      	'Australian Transport Safety Bureau',
+      	'Civil Aviation Safety Authority',
+      	'Moorebank Intermodal Company Limited',
+      	'National Capital Authority',
+      	'National Transport Commission'
+      ],
+      # 'Department of Parliamentary Services' => [],
+      # 'Department of the House of Representatives' => [],
+      # 'Department of the Senate' => [],
+      # 'Parliamentary Budget Office' => [],
+      'Department of the Prime Minister and Cabinet' => [
+      	'Aboriginal Hostels Limited',
+      	'Anindilyakwa Land Council',
+      	'Australian National Audit Office',
+      	'Australian Public Service Commission',
+      	'Central Land Council',
+      	'Digital Transformation Office',
+      	'Indigenous Business Australia',
+      	'Indigenous Land Corporation',
+      	'National Australia Day Council Limited',
+      	'Northern Land Council',
+      	'Office of National Assessments',
+      	'Office of the Commonwealth Ombudsman',
+      	'Office of the Inspector-General of Intelligence and Security',
+      	'Office of the Official Secretary to the Governor-General',
+      	'Outback Stores Pty Ltd',
+      	'Tiwi Land Council',
+      	'Torres Strait Regional Authority',
+      	'Wreck Bay Aboriginal Community Council'
+      ],
+      'Department of Social Services' => [
+      	'Australian Institute of Family Studies',
+      	'National Disability Insurance Agency',
+      	'Department of Human Services',
+      	'Australian Hearing Services'
+      ],
+      'Department of the Treasury' => [
+      	'Australian Bureau of Statistics',
+      	'Australian Competition and Consumer Commission',
+      	'Australian Office of Financial Management',
+      	'Australian Prudential Regulation Authority',
+      	'Australian Reinsurance Pool Corporation',
+      	'Australian Securities and Investments Commission',
+      	'Australian Taxation Office',
+      	'Commonwealth Grants Commission',
+      	'Corporations and Markets Advisory Committee',
+      	'Inspector-General of Taxation',
+      	'National Competition Council',
+      	'Office of the Auditing and Assurance Standards Board',
+      	'Office of the Australian Accounting Standards Board',
+      	'Productivity Commission',
+      	'Reserve Bank of Australia',
+      	'Royal Australian Mint'
+      ]
+    }
+
+    departments.each do |dept_name, agencies|
+
+      puts "** #{dept_name} **"
+
+      dept = Department.find_or_create_by!(name: dept_name)
+
+      agencies.each do |agency_name|
+        puts "** #{agency_name} **"
+        agency = Agency.find_or_create_by!(name: agency_name)
+        dept.sections << agency
```

Comment:
What about existing sections e.g. 'Communications' ? I can't see that in the lists ...